hibernate-core/src/main/java/org/hibernate/Session.java

```diff
@@ -1113,7 +1117,11 @@ public interface Session extends SharedSessionContract, EntityManager {
 	 * @return an instance of {@link IdentifierLoadAccess} for executing the lookup
 	 *
 	 * @throws HibernateException If the given name does not resolve to a mapped entity
+	 *
+	 * @deprecated This method will be removed.
+	 *             Use {@link #find(Class, Object, FindOption...)} instead.
 	 */
+	@Deprecated(since = "7.2", forRemoval = true)
```

Comment:
I think adding the deprecations to 7.1 is fine.
```suggestion
	@Deprecated(since = "7.1", forRemoval = true)
```